code/modules/mob/living/simple_animal/abnormality/aleph/staining_rose.dm

```diff
@@ -0,0 +1,143 @@
+#define STATUS_EFFECT_WILTING /datum/status_effect/wilting
+//Coded by me, Kirie Saito!
+/mob/living/simple_animal/hostile/abnormality/staining_rose
+	name = "Staining Rose"
+	desc = "A tiny, wilting rose."
+	icon = 'ModularTegustation/Teguicons/64x64.dmi'
+	icon_state = "rose"
+	maxHealth = 10
+	health = 10			//It's a rose lol
+	threat_level = ALEPH_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = 0,
+		ABNORMALITY_WORK_INSIGHT = list(0, 0, 0, 30, 40),
+		ABNORMALITY_WORK_ATTACHMENT = 0,
+		ABNORMALITY_WORK_REPRESSION = list(0, 0, 0, 45, 50)
+			)
+	start_qliphoth = 3
+	work_damage_amount = 16
+	work_damage_type = PALE_DAMAGE
+	pixel_x = -16
+	base_pixel_x = -16
+	pixel_y = -8
+	base_pixel_y = -8
+
+	ego_list = list(
+//		/datum/ego_datum/weapon/blooming,
+//		/datum/ego_datum/armor/blooming
+		)
+
+	var/chosen
+	var/meltdown_cooldown_time = 300 SECONDS
+	var/meltdown_cooldown
+
+
+/mob/living/simple_animal/hostile/abnormality/staining_rose/work_complete(mob/living/carbon/human/user, work_type, pe)
+	..()
+	if (chosen == null)
+		chosen = user
+		user.visible_message("<span class='warning'>You are now the rose's chosen.</span>")
+		icon_state = "rose_activated"
+
+	if (user != chosen)		//Your body starts to wilt.
+		user.visible_message("<span class='warning'>The rose has already chosen another, [chosen]!</span>")
+		user.physiology.red_mod *= 1.2
```

Comment:
All this stuff needs a timer to reset, otherwise it'll just force people to cryo & respawn.

Reply:
So
If you work on this you're either the chosen and already level 5, or you've got Level V repression, or you're already so fucked that a light breeze will blow you over. This is an aleph, the only difference is that instead of killing you it just permanently cripples you

Reply:
Quite an unfun mechanic, imho.

Reply:
Its a punishment for failure?
Better than most alephs just instakilling you under Level 5

Reply:
I do want to also mention that people ASKED me for this, instead of it just instantly killing you, like it apparently does in the comic